### DISCUSSION TITLE
7583 cannot build 7.50

### DISCUSSION THREAD

**Discussion by zackmuc** [Freetz-NG/freetz-ng#735] [Unerfüllte Wünsche]
Hello,

if I try to build for 7583 I get:

  checking kernel version
    ... used by AVM ...... 4.9.218
    ... used by Freetz ... 4.9.198

What do I have to do?

Regards

Peter

**Comment by pfichtner** [comment]
what's the error?

**Comment by fda77** [comment]
https://freetz-ng.github.io/freetz-ng/FIRMWARES.html

**Reply by zackmuc** [reply]
If i try to open I get an 404 :(

**Reply by kainers1** [reply]
https://github.com/Freetz-NG/freetz-ng/blob/master/docs/FIRMWARES.md

**Reply by fda77** [reply]
Geht wieder

**Comment by zackmuc** [comment]
processing kernel
  checking kernel version
    ... used by AVM ...... 4.9.218
    ... used by Freetz ... 4.9.198
ERROR: Freetz and AVM kernel versions do not match. Wrong kernel version configuration in Freetz?
Are GPL sources for YOUR device and firware version released at http://osp.avm.de/ yet?
Have YOU asked fritzbox_info@avm.de for sources?


on this page the 7.50 version is missing . In another forum someone saids this is the same kernel the 7590 uses ..

**Comment by fda77** [comment]
Im sure theres nothing "missing"
![image](https://user-images.githubusercontent.com/471850/218250483-4524bfbb-d053-4b35-b2f4-1be46b793c53.png)

**Comment by zackmuc** [comment]
But on the one side we have: 
![image](https://user-images.githubusercontent.com/4520343/218250960-35ceaa0f-accd-44f7-8d92-0f0906e6c4d2.png)
but here 
![image](https://user-images.githubusercontent.com/4520343/218250992-18302890-cf61-4743-b13b-0c1da5c35965.png)
and here  is already 7.50
![image](https://user-images.githubusercontent.com/4520343/218251041-80e7ac4e-6ed5-4046-9b85-ef7af5b8aeaf.png)

**Comment by fda77** [comment]
When it's not on the "Currently supported devices and firmwares" page, it means that it is currently not supported

**Comment by devmarxx** [comment]
It does build, when you use the 7590 kernel settings in the .config file:

https://github.com/Freetz-NG/freetz-ng/discussions/739

Regards

Chris
